src/components/Kv/KvCauseSelector.vue

```diff
@@ -45,7 +45,7 @@
 						alt=""
 					>
 				</div>
-				<div class="kv-cause-selector__text">{{ cause }}</div>
+				<div class="kv-cause-selector__text">{{ capitalizeFirstLetter(cause) }}</div>
```

Comment:
I believe you could also use: `{{ cause|changeCase('titleCase') }}`

Reply:
> I believe you could also use: `{{ cause|changeCase('titleCase') }}`

I think that would change 'COVID-19' to 'Covid-19' unfortunately

Reply:
Sign, I was afraid of that.

Reply:
@ryan-ludwig since all the causes are one word, we can just use CSS `text-transform: capitalize;`